diam/sm/smparser/cea.go

```diff
@@ -23,7 +23,7 @@ type CEA struct {
 }
 
 // Parse parses and validates the given message.
-func (cea *CEA) Parse(m *diam.Message) (err error) {
+func (cea *CEA) Parse(m *diam.Message, isServer uint8) (err error) {
```

Comment:
Same comments of `CER.Parse`.. either make a Role or offer `ParseClient` and `ParseServer` if possible.